.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request: ~
+  workflow_dispatch: ~
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js v16
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16.x"
+      - name: Install dependencies
+        run: npm i
+      - name: Build tokens
+        run: npm run build:tokens
+      - name: Build icons
+        run: npm run build:icons
+      - name: Formatting
+        run: npm run format
+      - name: Unit tests
+        run: npm run test
```

Comment:
i didn't see that we had a script for linting, but i could look into adding it

Reply:
we do need to lint in CI. you could either add it in this task or create a separate task for it. up to you.

it would essentially involve having a `lint` command on a per-project basis that runs appropriate linters for that project (eslint on TS/JS files and stylelint on (S)CSS files). then having a `lint` command at the root that runs the project `lint` commands via turbo